fairscale/nn/data_parallel/fully_sharded_data_parallel.py

```diff
@@ -172,11 +173,11 @@ def __init__(
         params = list(p for p in module.parameters() if not hasattr(p, "_is_sharded"))
 
         if self.flatten_parameters and len(params) > 0:
-            self.module: nn.Module = FlattenParamsWrapper(module, param_list=params)
+            self._fsdp_module: nn.Module = FlattenParamsWrapper(module, param_list=params)
```

Comment:
(nit) should this be called `_child_module/_wrapped_module/`? Does not seem guaranteed be an `FSDP` instance.

Reply:
yep, will rename to `_fsdp_wrapped_module`. Key is that it's easy to grep for in a state_dict, and unlikely to be in the user's module